de_web_plugin.cpp

```diff
@@ -5227,6 +5231,11 @@ void DeRestPluginPrivate::addSensorNode(const deCONZ::Node *node, const deCONZ::
                     {
                         fpBatterySensor.inClusters.push_back(ci->id());
                     }
+                   if (node->nodeDescriptor().manufacturerCode() == VENDOR_SI_LABS &&
```

Comment:
A space is missing before `if` :)

Reply:
Lol, a missing space on the 20k lines file ....

Done.